### ISSUE TITLE
stylelint showing CssSyntaxError on markdown files

### ISSUE THREAD

**Issue by nilsborg** [stylelint/stylelint#3212]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Today I noticed that stylelint in VSCode started to show errors inside markdown files. I think that's a bug …

> Which rule, if any, is this issue related to?

"Missing semicolon (CssSyntaxError)"

> What CSS is needed to reproduce this issue?

Well not CSS in this case but a markdown file with some front matter on the top. E.g.:

```
---
layout: faq
[more stuff]
---
```

> What stylelint configuration is needed to reproduce this issue?

I'm using this in my `package.json`:

```
"stylelint": {
  "extends": "stylelint-config-recommended-scss"
}
```
> Which version of stylelint are you using?

`9.1.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

VSCode Plugin version `0.34.0`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Well ... I guess. Since it's in markdown files :)

> What did you expect to happen?

That no linting would happen at all..

> What actually happened (e.g. what warnings or errors you are getting)?

For example this:

```shell
Missed semicolon (CssSyntaxError)
```

**Comment by jeddy3** [comment]
@nilsborg Thanks for the report and for using the template.

It looks like this problem was raised in [VSCode extension issue tracker](https://github.com/shinnn/vscode-stylelint/issues/148#issuecomment-371027295) and the author of that extension kindly raised an issue in postcss-html (a dependency of stylelint). The author of that package, in turn, fixed the issue. The chain broke down because we haven't released a new version of stylelint yet (mainly because the problem wasn't on our radar until you raised this issue).

@stylelint/core If no one has any objections, I can do a patch release tonight or tomorrow with the new `postcss-html` dependency.

**Comment by nilsborg** [comment]
Woohoooo Sounds Great 👍🏼

**Comment by jeddy3** [comment]
Released as `9.1.2`.
